### PR TITLE
doc: project organization and getting started

### PR DESCRIPTION
**Pull request by nicorikken**
To add labels and provide a guidance for getting started (sorry, somewhat of a combined PR).
Labels based on #105 

I was trying to go for some highlighting in accordance with the GitHub label colors. Unfortunately I couldn't easy find out how to do that, based on the Asciidoctor manual. Can I get some writing/coding advice?

### CODE REVIEW

**Comment by nicorikken** [comment]
Am I correct in saying that you can call `atom -d .` in the checkout out directory for development? Personally I have symlinked my checkout into `~/.atom/packages`

**Comment by mojavelinux** [comment]
> I was trying to go for some highlighting in accordance with the GitHub label colors.

You'll have to resort to images. GitHub does not allow the use of colorized text in an AsciiDoc file (or any markup language for that matter). One creative way to do it is to link to a badge service that lets you create an SVG with custom text and color.

For example:

**Comment by mojavelinux** [comment]
You still have to symlink it first.

```
apm link
atom -d .
```

**Comment by nicorikken** [comment]
> You'll have to resort to images. GitHub does not allow the use of colorized text in an AsciiDoc file

I'll leave this for now then.

**Comment by ldez** [comment]
`apm link -d` otherwise the package is bound even outside the dev context

**Comment by nicorikken** [comment]
Alright, changed it.

**Comment by nicorikken** [comment]
ready to merge?

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -5,6 +5,42 @@ In the spirit of free software, _everyone_ is encouraged to help improve this pr
...
+
+.Type labels
+|===
+|bug         | the current implementation has a flaw that needs to be resolved
+|question    | generic question
+|improvement | refinement on top of an existing feature
+|enhancement | new feature
+|duplicate   | a similar (open) issue already exsists
+|invalid     | not, or no longer, true
+|declined    | not accepted for inclusion
```
Comment (mojavelinux) [inline CONTRIBUTING.adoc]: I recommend changing the description of declined to "a decision was made not to pursue". That keeps the door open for someone who wants to look into it.

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -5,6 +5,42 @@ In the spirit of free software, _everyone_ is encouraged to help improve this pr
 To contribute code, simply fork the project on GitHub, hack away and send a pull request with your proposed changes.
 Feel free to use the https://github.com/asciidoctor/atom-language-asciidoc/issues[issue tracker] to provide feedback or suggestions in other ways.
 
+== Project organization
+
+The project is hosted and run using GitHub.
+The issue tracker is in place to report bugs, discuss improvements, show our ambitious goals and desires for the future, and encourage each other.
```
Comment (mojavelinux) [inline CONTRIBUTING.adoc]: I love the way you phrased this!

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -5,6 +5,42 @@ In the spirit of free software, _everyone_ is encouraged to help improve this pr
...
+Classification labels and versioning tags are added for clarity and the ability to search
```
Comment (mojavelinux) [inline CONTRIBUTING.adoc]: Missing a period?

File: CONTRIBUTING.adoc [hunk 4]
```diff
@@ -5,6 +5,42 @@ In the spirit of free software, _everyone_ is encouraged to help improve this pr
...
+Labels are added to issues and pull request for clarity, in the present but also when looking back at previous issues and pull requests.
+
+.Type labels
+|===
+|bug         | the current implementation has a flaw that needs to be resolved
+|question    | generic question
+|improvement | refinement on top of an existing feature
+|enhancement | new feature
+|duplicate   | a similar (open) issue already exsists
+|invalid     | not, or no longer, true
```
Comment (mojavelinux) [inline CONTRIBUTING.adoc]: For the description of "invalid", I recommend "not reproducible or out of date"

File: CONTRIBUTING.adoc [hunk 5]
```diff
@@ -27,7 +63,18 @@ apm install
...
-*todo*
+. Decide what you want to work on, and what should be the result
+. Have a GitHub account
+. Fork the project to your personal account
+. Check out your forked project for local development using git
+. Navigate to the directory using the command line interface
+. Link the package using `apm link -d`, and start atom in dev-mode using the command `atom -d .`
+. _Develop_ (this is where the magic happens)
+. Commit and push your code onto your forked project
+. Make a pull request to get the changes merged into the upstream project
```
Comment (mojavelinux) [inline CONTRIBUTING.adoc]: You might want to mention here that we use a pull request template so the contributor knows to expect it. Ask them to fill it out to the best of their abilities.

**Comment by mojavelinux** [comment]
I left some comments inline. Once you address those, I'm ready to endorse.

**Comment by nicorikken** [comment]
This latest commit should fix it. Please verify.

**Comment by mojavelinux** [comment]
Beautiful! :+1:

**Pull request merged**